validation_tests/dyninst/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+. ../../setup.sh
+spackLoadUniqueNoR dyninst
```

Comment:
Spack recently stopped recognizing the -r argument so packages are loaded recursively by default. If you are using NoR here because of an issue with the -r argument that should be fixed soon. Every spackLoadUnique operation is fairly time consuming so it's best to minimize redundant calls. I think just dyninst and cmake should be needed here.